addons/payment_adyen/utils.py

```diff
@@ -53,13 +53,19 @@ def format_partner_address(partner):
     """
     STREET_FORMAT = '%(street_number)s/%(street_number2)s %(street_name)s'
     street_data = split_street_with_params(partner.street, STREET_FORMAT)
+    if any(key not in street_data for key in ('street_name', 'street_name')):
+        raise ValidationError(_(
+            "wrong address format; received \"%s\" but expected \"%s\"",
+            partner.street, STREET_FORMAT.replace('%(', '').replace(')s', '')
+        ))
```

Comment:
You tried to combine both suggestions instead of only the second one, which doesn't need the ValidationError.
```suggestion
```